### ISSUE TITLE
Banner background is white instead of clear

### ISSUE THREAD

**Issue by vikdenic** [Zentrust/OTPublishersHeadlessSDK#26]
On **iOS 17**, there is an issue where the OneTrust banner background is white and blocks any views beneath it (SDK version **202309.1.0**). We are showing the banner in our app like so:

```
OTPublishersHeadlessSDK.shared.setupUI(viewController, UIType: .banner)
OTPublishersHeadlessSDK.shared.showBannerUI()
```

In the view debugger, you will see the Banner view-controller's view has a white background though:

<img width="1129" alt="view_debugger" src="https://github.com/Zentrust/OTPublishersHeadlessSDK/assets/7085697/96f0d4e1-cb28-4158-b42e-3e0f898e4c50">

This results in the views beneath being covered up (not desired):

<img width="275" src="https://github.com/Zentrust/OTPublishersHeadlessSDK/assets/7085697/507c822b-d2c2-432a-9666-d15060ccd725">

Is there a way to fix the OneTrust banner's background (which is supposed to be clear) from being white in iOS 17?

**Comment by techouse** [comment]
Same issue here. Works fine on iOS 16, but breaks in iOS 17.

**Comment by vikdenic** [comment]
Fixed in [v202311.1.0](https://developer.onetrust.com/onetrust/docs/changelog-ios-tvos#20231110)
